packages/desktop-electron/index.ts

```
@@ -204,6 +195,148 @@ async function createBackgroundProcess() {
+            logMessage('info', 'Sync-Server: Actual Sync Server has started!');
+            syncServerStarted = true;
+            resolve();
+            break;
+          default:
+            logMessage(
+              'info',
+              'Sync-Server: Unknown server message: ' + msg.type,
+            );
+        }
+      });
+    });
```

Biome (1.9.4)

[error] Promise executor functions should not be `async`.



(lint/suspicious/noAsyncPromiseExecutor)